1-Draft/RFCNNNN-RFC-Native-Command-Exit-Errors.md

```diff
@@ -0,0 +1,236 @@
+---
+RFC: RFCNNNN-RFC-Native-Command-Exit-Errors
+Author: Micheal Padden
```

Comment:
@mopadden is this the correct spelling of your name? If you'd prefer to use your GitHub moniker or similar, that might be acceptable too

Reply:
Yes, that's me Micheal Padden. Is it preferrable to use the name or the github moniker?